### PR TITLE
enable video previews on desktop app detail (bug 1066871)

### PR DESCRIPTION
**Pull request by spasovski**
To test this locally point your settings_local to stage and visit `/app/test-app-bat1429/`

### CODE REVIEW

File: src/media/css/image-deferrer.styl [hunk 1]
```diff
@@ -4,7 +4,7 @@
 img {
     transition: opacity .1s;
 }
-.screenshot .deferred {
+.thumbnail .deferred {
```
Comment (ngokevin) [inline src/media/css/image-deferrer.styl]: hmm, what more specific class name could we use? ```.thumbnail``` could easily collide down the road

File: src/media/js/previews.js [hunk 2]
```diff
@@ -39,6 +39,7 @@ define('previews',
     var mediaSwitch = '(min-width:' + dimensions.breakpoint + 'px)';
 
     var desktopSideMargin = 0;  // Keep track of tray offset.
+    var previewItemClass = '.thumbnail';
```
Comment (ngokevin) [inline src/media/js/previews.js]: nice, already predicting we're gonna nit over the class name

**Comment by ngokevin** [comment]
will apply this in a sec

**Comment by ngokevin** [comment]
looks great. could you also check it when you add maybe a couple more images or videos? test the scrolling action

r+wc

**Comment by Lord-of-Darkness** [comment]
Desarrolladores y líderes de Mozilla, que son travestis culo!

Alguien debe colgar ustedes para arriba en sus propias bolas y dejar que su perro muerda su caricatura de una polla!

Posteriormente deben ponerlo tiempo que sea necesario por la garganta hasta que traga a sí mismos en el suicidio!

Ustedes Mozilla está ... Estúpido de mierda coños!
--------------------------------------------------------
Developers and leaders of Mozilla, which are shemales ass!

Someone should hang you up in your own balls and let your dog bite his caricature of a cock!

They should then put it long as needed by the throat until he swallows themselves in suicide!

You Mozilla is... Stupid fucking pussies!

**Comment by Lord-of-Darkness** [comment]
Desarrolladores y líderes de Mozilla, que son travestis culo!

Alguien debe colgar ustedes para arriba en sus propias bolas y dejar que su perro muerda su caricatura de una polla!

Posteriormente deben ponerlo tiempo que sea necesario por la garganta hasta que traga a sí mismos en el suicidio!

Ustedes Mozilla está ... Estúpido de mierda coños!
--------------------------------------------------------
Developers and leaders of Mozilla, which are shemales ass!

Someone should hang you up in your own balls and let your dog bite his caricature of a cock!

They should then put it long as needed by the throat until he swallows themselves in suicide!

You Mozilla is... Stupid fucking pussies!

**Pull request merged**